libsoftwarecontainer/src/gateway/iptableentry.h

```diff
@@ -67,6 +67,18 @@ class IPTableEntry
      */
     ReturnCode applyRules();
 
+    /**
+     * @brief Interprets a rule to iptables applicable string
+     * @return string indicating interpreted rule
+     */
+    std::string interpretRule(Rule rule);
+
+    /**
+     * @brief Interprets defaultTarget rule to iptables applicable string
```

Comment:
What is "our syntax"? The GW configs? In that case, I'm not sure I like a config implementation detail to sneak its way to here. Perhaps it would be better to express it in more general terms so we don't have to remember updating this comment in case the config syntax changes slightly?